bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -449,6 +453,9 @@
 		<label>Filter Error</label>
 		<description>Filter full</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Alarm</tag>
```

Comment:
This sounds a bit too "serious" to me. It is more a "notification" that a consumable is used up. 
I can't find a more suitable tag for this, but I could see similar use cases in other appliances, e.g. filters in cleaning robots, where `Alarm` sounds even less suitable.

Reply:
Point taken. But I would interpret "alarm" as meaning an "off normal" state.

Reply:
But this is nothing that happens unpredictably but after a certain time of operation, which can be set by the user. So it is rather a `Status` message (maybe combined with proposed `Info` tag)